frontend/src/components/Header/index.js

```diff
@@ -58,7 +58,7 @@ const Header = ({ user, loadUser }) => {
         <>
           <NavLink to="/">Contul meu</NavLink>
           <div className="account-separator"></div>
-          <a onClick={handleLogout}>Logout</a>
+          <button onClick={handleLogout}>Logout</button>
```

Comment:
> @adascaliteiradu How do you access the `AddMember` component? Accessing [/add-member](https://stam-acasa-oa41xdak9.now.sh/add-member) doesn't seem to do it ...

You need to be authenticated, for any route in / to work

Reply:
There was no broken design, there was no design at all 😭. I have updated the styles for the buttons since the button is the best element to be placed there. It is not recommended to have that workaround (set "href" to #! and preventDefault) in place. Links should be used to navigate to other pages or to sections in the current page, not to trigger js logic.

Reply:
I don't understand the `there was no design` part ...

In [Figma](https://www.figma.com/file/rdg8gZtSWNYaVtupMtdhST/CoVid---Stam-Acasa?node-id=4%3A1) the account links looks like this:
<img width="242" alt="Screen Shot 2020-04-01 at 10 24 38 AM" src="https://user-images.githubusercontent.com/11520795/78110225-3ddf7500-7403-11ea-824d-8e5d1f9b7e5c.png">

In our app, now they look like this:
<img width="261" alt="Screen Shot 2020-04-01 at 10 22 39 AM" src="https://user-images.githubusercontent.com/11520795/78110249-48017380-7403-11ea-893e-2d1d1d222612.png">

Technically, the login button navigates to the Login section of the page 😃

Reply:
With the `there was no design` part I mean that when I first committed the buttons there were no styles applied to the buttons, so they looked like you said. In this [commit](https://github.com/code4romania/stam-acasa/pull/87/commits/10a743f206cca406a2cd71c1c897ec33411007af) I have created styles for the buttons so they look just like the links. I didn't see the gray background of the button, because it the default behavior for Chrome on Windows, but I fixed that issue in this [commit](https://github.com/code4romania/stam-acasa/pull/87/commits/b4ba9163ca64c616cfa11e011baec614d24ca2f4).

This is how they look like now:
![image](https://user-images.githubusercontent.com/17283534/78114582-2b1c6e80-740a-11ea-8bf2-a2ca6a3d25cb.png)

> Technically, the login __button__ navigates to the Login section of the page. 

Yea, but actually the `login` is not a section (#login or /login). The login button acts like a submit button in a login form (triggers the login OIDC logic or if the user is already authenticated, you don't even go to the login page). 
* `Contul meu` which navigates to a section on this page should be a link 
* `Inregistrare` which links to the external register page directly with no other js logic behind should be a link 
* `Login` and `Logout` trigger some js logic, without directly redirecting the user to the `login`/`logout` pages (this is a logic handled by OIDC), so they should be buttons
  * They do not redirect the user to: "#/"(`preventDefault`)?

What I'm trying to say is that we should not achieve the expected design using HTML elements, if we can create the styles using CSS 😄 .